android/src/toga_android/widgets/webview.py

```diff
@@ -86,3 +117,7 @@ def evaluate_javascript(self, javascript, on_result=None):
 
         self.native.evaluateJavascript(javascript, ReceiveString(result))
         return result
+
+    def set_on_navigation_starting(self, handler):
```

Comment:
At this point, I'm pretty sure we've removed all the "set_on_..." methods; unless there's a specific use case (and it doesn't look like there is), this can be removed.

Reply:
I removed it.